.travis.yml

```diff
@@ -19,6 +20,9 @@ install:
 - pip install -r requirements-dev.txt
 - pip install -e .[test]
 - pip list
+before_script:
+- pip install flake8  # 127 == GitHub editor width
+- if [[ $TRAVIS_PYTHON_VERSION != 2.6 ]]; then flake8 . --count --max-line-length=127 --statistics --exit-zero; fi
```

Comment:
Can we remove the flake8 check? Automatically checking code styles needs a bit more discussion in a dedicated ticket.